akamai/cache-client.go

```diff
@@ -160,6 +156,16 @@ func (cpc *CachePurgeClient) constructAuthHeader(request *http.Request, body []b
 	), nil
 }
 
+// signingKey makes a signing key by HMAC'ing the timestamp
+// using a client secret as the key.
+func signingKey(clientSecret string, timestamp string) []byte {
```

Comment:
I don't think this comment is right (looks inherited from the old code, so not your bad!). It doesn't make a signing key it makes an HMAC tag over the timestamp using the `clientSecret` as the key. I think `key` should be renamed to `tag` or `hash` and the comment updated.

Reply:
I missed the second HMAC part. Akamai what are you doing..............

In that case the comment is right to refer to it as a key and I withdraw my complaints. I don't feel strongly about calling it a "signing" key or not.